src/admin/WL_Metabox/WL_Metabox_Field_date.php

```diff
@@ -2,14 +2,37 @@
 
 
 class WL_Metabox_Field_date extends WL_Metabox_Field {
+	
+	/*
+	 * Attribute to distinguish between date formats, inferred from the schema property export type
+	 * 
+	 * @since 3.2.0
+	 */
```

Comment:
@pieroit add here the details about the variable:

    * @access private
    * @var string $date_format The date format.